food-bank-app/src/components/svg/LargeEclipseSvg.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Svg, { SvgProps, Circle } from "react-native-svg"
+
+export const LargeEclipseSvg = (props: SvgProps) => {
```

Comment:
normalmente los acronimos pueden ser excepciones a camelcase 
```suggestion
export const LargeEclipseSVG = (props: SVGProps) => {
```

Reply:
lmao, ya que el paquete lo escribe así. Nvm entonces